src/docs/MLOS/references/Overview.adoc

```diff
@@ -1,9 +1,13 @@
-*Machine Learning Open Studio (ML-OS)* is an interactive graphical interface that enables developers and data scientists to quickly and easily build, train, and deploy machine learning models at any scale. It provides a rich set of generic machine learning tasks that can be connected together to build either basic or complex machine learning workflows for various use cases such as: fraud detection, text analysis, online offer recommendations, prediction of equipment failures, facial expression analysis, etc.
+*Proactive Machine Learning (PML)* is a complete DSML platform (Data Science and Machine Learning) including a ML Studio, AutoML, Data Science Orchestration and MLOps for the deployment,
+training, execution and scalability of artificial intelligence and machine learning models on
+any type of infrastructure. Created for data scientists and ML engineers, the solution is simple to use and accelerate the development and deployment of machine learning models.
 
-image::ML_OS_overview.png[align=center]
-Machine Learning Open Studio is open source and allows easy task parallelization, running them on resources matching constraints (Multi-CPU, GPU, data locality, library).
+
+image::PML_overview.png[align=center]
+Proactive Machine Learning platform provides a rich catalog of generic machine learning tasks that can be connected together to build either basic or advanced machine learning workflows for various use cases such as: fraud detection, text analysis, online offer recommendations, prediction of equipment failures, facial expression analysis, etc.
+PML workflows enable users to manage machine learning pipelines through the different phases of the development lifecycle and allow them to better control tasks parallelization, by running the tasks on resources matching constraints (Multi-CPU, GPU, FPGA, data locality, libraries, etc).
 
 
 image::ML-Open-Studio-ActiveEon.png[align=center]
 
-The Machine Learning Open Studio  can be tried for free online https://try.activeeon.com/studio/#workflows/templates/machine-learning[here^].
+The Proactive Machine Learning platform is an open source solution, and it can be tested online without installation on our try platforms.
```

Comment:
can you link try to https://try.activeeon.com/studio/#workflows/templates/machine-learning ?

Reply:
done, thank you @aminelouati :)